lib/customer.rb

```diff
@@ -0,0 +1,47 @@
+class Customer
+
+  attr_accessor :order, :total_price, :total_quantity
+  attr_reader :summary
+
+  def initialize
+    @order = {}
+    @total_price = 0
+    @total_quantity = 0
+    # @summary = [["Item","Quantity","Price"]]
+  end
+
+  def check_menu(menu)
+    menu.view_menu
+  end
+
+  def add_to_order(menu,food,quantity)
```

Comment:
Assignment Branch Condition size for add_to_order is too high. [15.07/15]<br>Space missing after comma.